Makefile

```diff
@@ -64,7 +64,25 @@ run-local:
 	./meshery; \
 	cd ..
 
+run-tests:
+	# -----Linting Check-----
+	# GOPROXY=direct GOSUMDB=off go get github.com/mgechev/revive
+	revive -config tools-config/revive-lint.toml -formatter friendly ./... \
 
+	# ------Error Check------
+	# GOPROXY=direct GOSUMDB=off GO111MODULE=on go get github.com/kisielk/errcheck
+	errcheck -tags draft ./... \
+
+	# ------Static Check------
+	# GOPROXY=direct GOSUMDB=off GO111MODULE=on go get honnef.co/go/tools/cmd/staticcheck
+	staticcheck -tags draft -checks all,-ST1003,-ST1000,-U1000 ./... \
```

Comment:
Make sure this is working, as there can be a case where I have not configured my GOPATH.